### PR TITLE
ci: don't exceed max ids to terminate instances

### PR DESCRIPTION
**Pull request by jmdeal**
Fixes #N/A 

**Description**
Fixes an issue with the sweeper where if more than 1000 instances are leaked, it fails to clean up the instances since the id count exceeds the maximum that can be provided to terminate-instances.

**How was this change tested?**

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [x] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jigisha620 — DISMISSED** [review]
LGTM

File: test/hack/resource/pkg/resourcetypes/instance.go [hunk 1]
```diff
@@ -120,12 +121,20 @@ func (i *Instance) Get(ctx context.Context, clusterName string) (ids []string, e
...
 // Cleanup any old instances that were managed by Karpenter or were provisioned as part of testing
 func (i *Instance) Cleanup(ctx context.Context, ids []string) ([]string, error) {
-	if _, err := i.ec2Client.TerminateInstances(ctx, &ec2.TerminateInstancesInput{
-		InstanceIds: ids,
-	}); err != nil {
-		return nil, err
+	// The maximum number of EC2 instances which can be specified in a single API call
+	const maxIDCount = 1000
+	cleaned := make([]string, 0, len(ids))
+	errs := make([]error, 0, len(ids)/maxIDCount+1)
```
Comment (jonathan-innis) [inline test/hack/resource/pkg/resourcetypes/instance.go:127]: nit: Just to make this logic a little easier to parse -- does it make sense to chunk the list first and then just take the length of that to make the error slice?

**Review by jonathan-innis — DISMISSED** [review]

File: test/hack/resource/pkg/resourcetypes/instance.go (hunk 1 above)
Comment (jmdeal) [reply test/hack/resource/pkg/resourcetypes/instance.go]: Sure, seems fine to me. I'll make the update.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**